resources/js/components/cp/image-resize/ResizeForm.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+    <div>
+      <div class="flex items-center justify-between mb-3">
+        <h1 v-text="title"></h1>
+        <div>
+          <button class="btn-primary" :disabled="checkJobs || !checkAllDisabled" v-on:click="onTriggerResizeImages">Optimize remaining images</button>
+          <button class="btn-primary" :disabled="checkJobs" v-on:click="onTriggerResizeImages(true)">Optimize all images</button>
+        </div>
+      </div>
+
+      <div v-show="!loadingMessage" class="mt-2">
+          <ul class="card p-0 mb-2">
+              <li class="flex items-center justify-between py-1 px-2 border-b group">
+                <p>
```

Comment:
I think you can leave out the p tag. Semantically it's not incorrect but the li should be sufficient.